packages/xchain-thorchain/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v0.24.0 (2022-03-15)
```

Comment:
Set date to `2022-xx-xx`. We will start to publish a `xchain-thorchain@0.24.0-alpha.1` release and a final version later.